### PR TITLE
First impression on Nezha RISC-V SBC

### PR DESCRIPTION
**Pull request by Cezarus27**

### CODE REVIEW

File: blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md [hunk 1]
```diff
@@ -0,0 +1,941 @@
...
+## Intro
+
+Nezha board is a development board that is designed by an AWOL. This project
+uses a D1 SoC from Allwinner which is used for the first time by the general
+public. Probably this board is the **first massive** produced and available SBC
+based at RISC-V architecture taking in mind a fact that [BeagleV™](https://blog.3mdeb.com/2021/2021-05-06-first-impressions-beaglev/)
+was canceled in august this year after the pilot program with version beta of
+the board.
+
+This article is such an opening to the series of posts about Neza D1 where
```
Comment (TomaszAIR) [inline blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md:40]: @Cezarus27 I am not sure if there will be a series of posts, let's explain here how the board can be acquired

File: blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md [hunk 2]
```diff
@@ -0,0 +1,941 @@
...
+USB-UART converter, 2xUSB type-c cables, and screws.
+
+![nezha unboxing 1](/img/nezha-unboxing-1.jpg)
+
+![nezha unboxing 2](/img/nezha-unboxing-2.jpg)
+
+## Firmware & Operating System
+
+For now, the D1 Nezha development board comes with `Tina` Linux system which is
+a fork of the `OpenWRT`. It supports kernels such as Linux3.4, Linux3.10,
```
Comment (TomaszAIR) [inline blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md:78]: @Cezarus27 some link to the OpenWRT system will be great.

File: blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md [hunk 3]
```diff
@@ -0,0 +1,941 @@
...
+wiki.
+
+## First boot
+
+As it is described in many posts and comments on the entire internet, the board
+was shipped with `Tina Linux` installed at NAND. Below you can see bootlog from
+this version:
+
+```shell
+[188]HELLO! BOOT0 is starting!
```
Comment (TomaszAIR) [inline blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md:105]: @Cezarus27 asciinema would be better than added log

File: blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md [hunk 4]
```diff
@@ -0,0 +1,941 @@
...
+
+![nezha tina](/img/nezha-tina.jpg)
+
+## Booting Debian image
+
+There are two available distros of Debian. In this section let's take a look at
+one of them prepared by the **PerfXLab**.
+
+```shell
+[104]HELLO! BOOT0 is starting!
```
Comment (TomaszAIR) [inline blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md:724]: @Cezarus27 also here asciinema would be better

File: blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md [hunk 5]
```diff
@@ -0,0 +1,941 @@
...
+For now, the D1 Nezha development board comes with `Tina` Linux system which is
+a fork of the `OpenWRT`. It supports kernels such as Linux3.4, Linux3.10,
+Linux4.4, Linux4.9, Linux5.4, and others. There are also other distributions
+available such as Debian (`Sipeed` and `PerfXLab` versions) and Fedora. All of
+them can be found and download [here](https://ovsienko.info/D1/).
+
+On the other hand boot firmware on D1 consists of three parts, which largely
+correspond to the components used by 64-bit ARM SoCs:
+
+* `boot0` - it is modified for this board and used as SPL due to features such
```
Comment (TomaszAIR) [inline blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md:86]: @Cezarus27 something more about the boot flow? Maybe some diagrams?

File: blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md [hunk 6]
```diff
@@ -0,0 +1,348 @@
...
+[here](https://d1.docs.aw-ol.com/en/study/study_1tina/).
+It supports kernels such as Linux3.4, Linux3.10, Linux4.4, Linux4.9, Linux5.4,
+and others. There are also other distributions available such as Debian
+(`Sipeed` and `PerfXLab` versions) and Fedora. All of them can be found and
+download [here](https://ovsienko.info/D1/).
+
+On the other hand boot firmware on D1 consists of three parts, which largely
+correspond to the components used by 64-bit ARM SoCs:
+
+![nezha boot flow](/img/nezha-boot-flow.png)
```
Comment (TomaszAIR) [inline blog/content/post/2021-10-14-nezha-riscv-sbc-first-impression.md:91]: @Cezarus27 do this display correctly? Diagram is not added

**Review by artur-rs — APPROVED** [review]

**Pull request merged**